src/main/java/com/gdschongik/gdsc/domain/study/dto/response/StudentMyCompleteStudyResponse.java

```diff
@@ -0,0 +1,38 @@
+package com.gdschongik.gdsc.domain.study.dto.response;
+
+import com.gdschongik.gdsc.domain.common.model.SemesterType;
+import com.gdschongik.gdsc.domain.study.domain.AchievementType;
+import com.gdschongik.gdsc.domain.study.domain.StudyHistory;
+import com.gdschongik.gdsc.domain.study.domain.StudyHistoryStatus;
+import io.swagger.v3.oas.annotations.media.Schema;
+import java.util.ArrayList;
+import java.util.List;
+
+public record StudentMyCompleteStudyResponse(
+        Long studyId,
+        @Schema(description = "학년도") Integer academicYear,
+        @Schema(description = "학기") SemesterType semesterType,
+        @Schema(description = "이름") String title,
+        @Schema(description = "종류") String studyType,
+        @Schema(description = "상세설명 노션 링크") String notionLink,
+        @Schema(description = "한 줄 소개") String introduction,
+        @Schema(description = "멘토 이름") String mentorName,
+        @Schema(description = "총 주차수") Long totalWeek,
+        @Schema(description = "수료 여부") StudyHistoryStatus studyHistoryStatus,
+        @Schema(description = "우수 스터디원 여부") List<AchievementType> achievements) {
```

Comment:
널러블 어노테이션도 달아주면 좋을 것 같아요~
```suggestion
        @Nullable @Schema(description = "우수 스터디원 여부") List<AchievementType> achievements) {
```